src/me/sheimi/sgit/dialogs/CloneDialog.java

```diff
@@ -115,19 +129,25 @@ public void onClick(View view) {
             mRemoteURL.requestFocus();
             return;
         }
-        if (localPath.equals("")) {
-            showToastMessage(R.string.alert_localpath_required);
-            mLocalPath.setError(getString(R.string.alert_localpath_required));
-            mLocalPath.requestFocus();
-            return;
-        }
+        if (localPath.equals(""))
+            if (mLocalPath.getHint().equals(getString(R.string.dialog_clone_local_path_hint))) {
+                showToastMessage(R.string.alert_localpath_required);
+                mLocalPath.setError(getString(R.string.alert_localpath_required));
+                mLocalPath.requestFocus();
+                return;
+            }
         if (localPath.contains("/")) {
             showToastMessage(R.string.alert_localpath_format);
             mLocalPath.setError(getString(R.string.alert_localpath_format));
             mLocalPath.requestFocus();
             return;
         }
 
+        // If user is accepting the default path in the hint, we need to set localPath to
```

Comment:
This block could probably be moved to the initialisation point of ```localPath``` and thus negate the need for the added check on ```mLocalPath``` in the previous hunk.  I will fix that later tonight and ensure the behaviour is correct.